### DISCUSSION TITLE
Fine-tuning AraBERT for Q/A (self-supervised)

### DISCUSSION THREAD

**Discussion by ameenaltajer** [aub-mind/arabert#170] [Q&A] [answered]
Hello guys,

I wanted to know if it's possible to fine-tune AraBERT (for Q/A) in a self-supervised way, I want to supply the Arabic corpus without providing the labeled answers, I'm trying to validate the ability to use online learning for dynamic content Q/A scenarios without any supervision from my end.

Is AraBERT capable of doing so? Any thoughts here would be fantastic!

Thanks guys for the superb work :)

Best,
Ameen

**Comment by WissamAntoun** [comment, marked as answer]
Hey,
Also this might be doable, I've never heard of an approach for unsupervised/self-supervised QA. Which doesn't mean it doesn't exist.

If you ever find something that works on english BERT then it should definitely work with AraBERT.

Good Luck

**Reply by ameenaltajer** [reply]
Thank you very much, Wissam :)
